### PR TITLE
[KRK-2020] Kraków site update, welcome page, program, sponsors

### PR DESCRIPTION
**Pull request by jsuchenia**
Update of Kraków 2020 page with recent content. Removed CFP, update venue, sponsors..

### CODE REVIEW

File: content/events/2020-krakow/sponsor.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Type = "event"
 Description = "Sponsor devopsdays Kraków 2020"
 +++
 
-We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [<a href="mailto:karolina.murdza@proidea.pl">karolina.murdza@proidea.pl</a>].
```
Comment (bridgetkromhout) [inline content/events/2020-krakow/sponsor.md]: The email address you publish needs to be your devopsdays.org one, so the core team can stay in the loop.

File: content/events/2020-krakow/welcome.md [hunk 2]
```diff
@@ -5,70 +5,97 @@ aliases = ["/events/2020-krakow/"]
...
+  <div class='text'>
+   DevOpsDays Kraków 2020 is the first southern Poland edition of this volunteer-driven worldwide series of events.
+   It will take place on April 21st at the Variete Theatre. We are excited to bring this technical conference covering topics of Cloud Native, Containers and Microservices, Lean and DevOps approach to the heart of Europe. We expect 300+ IT professionals and enthusiasts eager to learn from each other’s experiences. We are looking forward to meeting you! Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection of them. It’s both a technical conference and a conference focusing on culture, processes, and structure within organizations. We encourage both technologists and business people to attend, learn and share experiences. Come to Kraków and join us to share this big DevOps adventure!
+  </div>
+  <div class="text">
+    <a href="https://evenea.pl/event/dodkrakow2020/" type="button" class="btn btn-danger btn-lg btn-block">REGISTRATION</a>
+    <a href="mailto:karolina.murdza@proidea.pl" type="button" class="btn btn-danger btn-lg btn-block">SPONSOR</a>
```
Comment (bridgetkromhout) [inline content/events/2020-krakow/welcome.md]: Again, you may not publish contact info for sponsors to use that does not go through krakow@devopsdays.org.

File: content/events/2020-krakow/sponsor.md (hunk 1 above)
Comment (jsuchenia) [reply content/events/2020-krakow/sponsor.md]: @bridgetkromhout Thank you - updated to our contact page

File: content/events/2020-krakow/welcome.md (hunk 2 above)
Comment (jsuchenia) [reply content/events/2020-krakow/welcome.md]: @bridgetkromhout as above - updated

**Pull request merged**